store/schemagraph.go

```diff
@@ -26,6 +60,7 @@ func (g *schemaGraph) traverse(fnVisit func(node *schemaNode) error) error {
 	return nil
 }
 
+// visitSchemaNode ???
```

Comment:
visitSchemaNode is used by traverse to make sure a node is only "visited" once (the callback function is called only once for each node)